src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,25 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import usersFromServer from '../../api/users';
+import { Todo } from '../../App';
+import { User } from '../../App';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map((todo, index) => {
+        const { userId } = todo;
+        const currentUser: User = usersFromServer.find(
+          user => user.id === userId,
+        )!;
```

Comment:
Using the non-null assertion operator (`!`) assumes that a user will always be found for each todo. This could lead to runtime errors if a user is not found. Consider handling the case where `find` returns `undefined`, possibly by adding a fallback or error handling mechanism.